.travis.yml

```diff
@@ -4,7 +4,7 @@
 sudo: required
 language: go
 go:
-- 1.8
+- 1.9
```

Comment:
what about using `1.11` ?

Reply:
1.11 will fail at this time. Still need to check what needs to be changed for higher version.